doc/developer-guides/hld/hld-virtio-devices.rst

```diff
@@ -547,8 +448,7 @@ VBS APIs
 ========
 
 The VBS APIs are exported by VBS related modules, including VBS, DM, and
-Service VM kernel modules. They can be classified into VBS-U and VBS-K APIs
-listed as follows.
+Service VM kernel modules.
```

Comment:
Are there still kernel modules providing VBS APIs now that we do not have VBS-K anymore?